.github/ISSUE_TEMPLATE/bugReport.yml

```diff
@@ -0,0 +1,36 @@
+name: "Bug Report"
+description: "Encountered a bug? Report it here"
+title: "Bug"
```

Comment:
```suggestion

```
Suggest removing the default title, as this allows users to submit reports with generic titles.